live-examples/html-examples/meta.json

```diff
@@ -1,13 +1,21 @@
 {
     "pages": {
+        "abbr": {
+            "baseTmpl": "tmpl/live-tabbed-tmpl.html",
+            "exampleCode": "live-examples/html-examples/abbr.html",
+            "cssExampleSrc": "live-examples/html-examples/css/abbr.css",
+            "fileName": "abbr.html",
+            "title": "HTML Demo: &lt;abbr&gt;",
+            "type": "tabbed"
+        },
         "audio": {
             "baseTmpl": "tmpl/live-tabbed-tmpl.html",
             "exampleCode": "live-examples/html-examples/audio.html",
             "cssExampleSrc": "live-examples/html-examples/css/audio.css",
             "fileName": "audio.html",
             "title": "HTML Demo: audio",
             "type": "tabbed"
-      },
+        },
```

Comment:
Nit: unintended indent?

Reply:
No, I think this is fixing a previous indent error, isn't it? (see the current master: https://github.com/mdn/interactive-examples/blob/master/live-examples/html-examples/meta.json)